bin/build-push-or-up.sh

```diff
@@ -3,7 +3,11 @@
 set -e
 
 export REGISTRY_PREFIX=${REGISTRY_PREFIX:-""}
-export DEFAULT_TAG=`git log --pretty=format:'%h' -n 1`
+export DEFAULT_TAG=`git log --pretty=format:'%h' -n 1 2>/dev/null`
+if [ "x$DEFAULT_TAG" = "x" ]
+then
+    DEFAULT_TAG="emptytag"
```

Comment:
export DEFAULT_TAG=`git log --pretty=format:'%h' -n 1 2>/dev/null` will set DEFAULT_TAG="" maybe in case of *git command no found* . To fix it, I give a DEFAULT_TAG a default value *emptytag*